build.sh

```diff
@@ -34,26 +34,27 @@ _setup_toolchain () {
 	TARGET_ABI="$1"
 	if [ "$TARGET_ABI" == armeabi-v7a ]; then
 		API=16
-		CROSS_PREFIX=arm-linux-androideabi
-		export CC=armv7a-linux-androideabi$API-clang
-		export CXX=armv7a-linux-androideabi$API-clang++
+		CROSS_PREFIX=armv7a-linux-androideabi
 		CFLAGS="-mthumb"
 		CXXFLAGS="-mthumb"
 	elif [ "$TARGET_ABI" == arm64-v8a ]; then
 		API=21
 		CROSS_PREFIX=aarch64-linux-android
-		export CC=$CROSS_PREFIX$API-clang
-		export CXX=$CROSS_PREFIX$API-clang++
 	elif [ "$TARGET_ABI" == x86 ]; then
 		API=16
 		CROSS_PREFIX=i686-linux-android
-		export CC=$CROSS_PREFIX$API-clang
-		export CXX=$CROSS_PREFIX$API-clang++
+		CFLAGS="-mssse3 -mfpmath=sse"
+		CXXFLAGS="-mssse3 -mfpmath=sse"
+	elif [ "$TARGET_ABI" == x86_64 ]; then
+		API=21
+		CROSS_PREFIX=x86_64-linux-android
 		CFLAGS="-mssse3 -mfpmath=sse"
 		CXXFLAGS="-mssse3 -mfpmath=sse"
```

Comment:
not needed on x86-64

Reply:
Why do we need it in the first place?

Reply:
The source for these was probably the same issue as this https://github.com/minetest/irrlicht/pull/83